homeassistant/components/media_player/panasonic_viera.py

```diff
@@ -183,3 +185,17 @@ def media_next_track(self):
     def media_previous_track(self):
         """Send the previous track command."""
         self.send_key('NRC_REW-ONOFF')
+
+    def play_media(self, media_type, media_id, **kwargs):
+        """Play media."""
+        _LOGGER.debug("Play media: %s (%s)", media_id, media_type)
+
+        if media_type == MEDIA_TYPE_URL:
+            try:
+                self._remote.open_webpage(media_id)
+            except (TimeoutError, OSError):
+                self._state = STATE_OFF
```

Comment:
You want potentially add an else statement here with some logging in case `media_type` is something else than expected, I think a warning would do fine to let someone looking into the logs to see what is happening.

Reply:
Done!